analysisCode/Makefile

```diff
@@ -33,7 +33,7 @@ CXXFLAGS += $(ROOTCFLAGS) -I$(HEADRDIR) $(LIBS)
 CXXFLAGS += -Wno-unused-variable -Wno-sign-compare
 
 # to add a class just add the corresponding .o library here
-MODULES = src/TruthEvent.o src/SmearedEvent.o src/EventLoop.o 
+MODULES = src/EventLoop.o src/TruthEvent.o src/SmearedEvent.o src/BreitFrame.o
```

Comment:
Let's leave src/EventLoop.o last in this last, so that the classes it depends on are compiled first.

Reply:
Will do, thanks for the pointer, I still have a fair amount to learn about Makefiles.